src/main/java/reposense/system/LogsManager.java

```diff
@@ -0,0 +1,99 @@
+package reposense.system;
+
+import java.io.IOException;
+import java.util.logging.ConsoleHandler;
+import java.util.logging.FileHandler;
+import java.util.logging.Handler;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.logging.SimpleFormatter;
+
+/**
+ * Configures and manages the loggers and handlers, including their levels
+ */
+public class LogsManager {
+    private static final int FILE_COUNT = 2;
+    private static final int MEGABYTE = (int) (Math.pow(2.0, 20));
+    private static final int MAX_FILE_SIZE_IN_BYTES = 5 * MEGABYTE; // 5MB
+    private static final String LOG_FILE = "reposense.log";
+    private static Level currentConsoleLogLevel = Level.INFO;
```

Comment:
should have an empty line to separate the `static final` from `static` variables (not idea how this passed checkstyle)